hadoop-hdds/docs/content/feature/Short-Circuit-Read.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Short Circuit Local Read in Datanode"
+weight: 2
+menu:
+   main:
+      parent: Features
+summary: Introduction to Ozone Datanode Short Circuit Local Read Feature
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Current in Ozone, client reads data over GRPC from Datanode. When the client asks the DataNode to read a file, the DataNode reads that file off of the disk and sends the data to the client over GRPC connection.
+
+This “short-circuit” local read feature will bypass the DataNode, allowing the client to read the file from local disk directly when the client is co-located with the data on the same server.
+
+Short-circuit local read can provide a substantial performance boost to many applications, by removing the overhead of network communication. 
+  
+## Prerequisite
+
+Short-circuit local reads make use of a UNIX domain socket. This is a special path in the filesystem that allows the client and the DataNodes to communicate.
+
+The Hadoop native library "libhadoop.so" provides the support to use the Unix domain socket. Please refer to Native Libraries ("https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/NativeLibraries.html") for details of this library.
+
+Before enabling short-circuit local reads, you must have a proper libhadoop.so, and make sure it's under the directory where Java can find and load it through "System.loadLibrary()" call.
```

Comment:
I'm not sure what is meant by a "proper" libhadoop.so. These three mini paragraphs all address the same topic and can be grouped into one paragraph. Additionally we should have instructions for how to make java see the library before talking about loading it.